docs/Rest-API.md

```diff
@@ -0,0 +1,147 @@
+# Axe REST API
+
+Documentation of the REST API that comes along with the Axe service.
+
+## AxeResult
+
+This endpoint serves as a wrapper to collect all of the information from an axe run. 
+
+### Motivation
+
+- Provide access to results and context easily.
+- Avoid timing issues that come with multiple requests.
+
+### Simplified Sample
+```
+{
+  "axeContext" : { @AxeContext },
+  "axeResult" : { AXRL },
```

Comment:
Does AxeResult include information on what app was scanned? I didn't find that anywhere in this doc. How about a link to the AXRL doc?

Reply:
AxeContext includes this information. A link to the AXRL doc is a good idea. I do not actually plan on hosting this here, I think it belongs elsewhere, and so this doc is inherently mildly incomplete.

Reply:
Absolutely agree with Rob that this document is significantly incomplete without more information about the AXRL format

---
In reply to: [274424144](https://github.com/dequelabs/axe-android/pull/11#discussion_r274424144) [](ancestors = 274424144)